README.md

```diff
@@ -24,6 +24,9 @@ This repo contains samples for Azure Data Manager for Energy APIs.
 * [Workflow Service](/adme-samples/rest-apis/index.html?page=/adme-samples/rest-apis/M18/ingestion_worflow_openapi.yaml)
 * [Secret Service](/adme-samples/rest-apis/index.html?page=/adme-samples/rest-apis/M18/secret_openapi.yaml)
 
+**Note:**
+Partition service's swagger is not uploaded to this repo because it is an internal service. Users are not allowed to call the partition service directly. Partition service is called by other services indirectly for reading partition info; It's also invoked when users interact with the partition management section of ADME Azure portal experience and the ADME partition management APIs.
```

Comment:
Good to modify this statement because as of now users can call into partition service, there is no layer which will block their call, this behavior will soon be changed in the future, till now it is recommended by ADME team to not call partition service

Reply:
removed